### PR TITLE
Add new profile 2018-03-01-hybrid

### PR DESCRIPTION
**Pull request by viananth**

### CODE REVIEW

File: azure-common/azure/profiles/__init__.py [hunk 1]
```diff
@@ -108,6 +108,45 @@ class KnownProfiles(Enum):
         },
         "2017-03-09-profile"
     )
+    v2018_03_01_hybrid = ProfileDefinition(
+        {
+            "azure.keyvault.KeyVaultClient":{
```
Comment (bganapa) [inline azure-common/azure/profiles/__init__.py]: should we include storage data plane here, though we don't have a clean story data plane at the ARM level, we are keeping the support at the PS/CLI tools . Go/Ruby/C#  SDKs right now does not include data plane (I need to check java)

File: azure-common/azure/profiles/__init__.py [hunk 2]
```diff
@@ -108,6 +108,45 @@ class KnownProfiles(Enum):
...
+            "azure.mgmt.authorization.AuthorizationManagementClient": {
+                None: "2015-07-01"
+            },
+            "azure.mgmt.compute.ComputeManagementClient": {
+                None: "2017-03-30"
+            },
+            "azure.mgmt.keyvault.KeyVaultManagementClient":{
+                None: "2016-10-01"
+            },
+            "azure.mgmt.network.NetworkManagementClient": {
```
Comment (bganapa) [inline azure-common/azure/profiles/__init__.py]: there should be one for DNS here

File: azure-common/azure/profiles/__init__.py [hunk 3]
```diff
@@ -108,6 +108,45 @@ class KnownProfiles(Enum):
...
+                None: "2016-09-01"
+            },
+            "azure.mgmt.resource.resources.ResourceManagementClient": {
+                None: "2018-02-01"
+            },
+            "azure.mgmt.resource.subscriptions.SubscriptionClient": {
+                None: "2016-06-01"
+            }
+        },
+        "2018-03-01-hybrid"
```
Comment (bganapa) [inline azure-common/azure/profiles/__init__.py]: do we have any SDK level tests that needs to be updated?

File: azure-common/azure/profiles/__init__.py [hunk 4]
```diff
@@ -142,6 +142,15 @@ class KnownProfiles(Enum):
             },
             "azure.mgmt.resource.subscriptions.SubscriptionClient": {
                 None: "2016-06-01"
+            },
+            "azure.mgmt.dns": {
+                None: "2016-04-01"
+            },
+            "azure.multiapi.storage": {
```
Comment (lmazuel) [inline azure-common/azure/profiles/__init__.py]: This is a CLI thing and does not exist at the SDK level.

File: azure-common/azure/profiles/__init__.py [hunk 5]
```diff
@@ -142,6 +142,15 @@ class KnownProfiles(Enum):
...
+                None: "2017-04-17"
+            },
+            "azure.multiapi.cosmosdb": {
```
Comment (lmazuel) [inline azure-common/azure/profiles/__init__.py]: This is a CLI thing and does not exist at the SDK level.

File: azure-common/azure/profiles/__init__.py (hunk 3 above)
Comment (lmazuel) [reply azure-common/azure/profiles/__init__.py]: I don't have profile specific tests unfortunately :(

**Pull request merged**